src/libraries/System.IO.Compression.Brotli/tests/System.IO.Compression.Brotli.Tests.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configurations>$(NetCoreAppCurrent)-Unix-Debug;$(NetCoreAppCurrent)-Unix-Release;$(NetCoreAppCurrent)-Windows_NT-Debug;$(NetCoreAppCurrent)-Windows_NT-Release</Configurations>
+    <DefineConstants Condition="'$(TargetsNetCoreApp)' == 'true'">$(DefineConstants);STREAM_MEMORY_OVERLOADS_AVAILABLE</DefineConstants>
```

Comment:
where is that define coming from?

Reply:
side effect of merge conflict.  It got removed in https://github.com/dotnet/runtime/commit/5afe409d59460195955b85882df41df0218a18b5#diff-df83858fafe4990170a741b9ea944f0aL4